vdj_ann/src/annotate.rs

```diff
@@ -2707,10 +2721,12 @@ pub fn cdr3_loc<'a>(
     }
     let mut i = ann.len() - 1;
     loop {
-        let t = ann[i].2 as usize;
```

Comment:
I think this is a case where binding a local is a lot more readable, we might want to restore this.